store/src/java/com/zimbra/cs/service/admin/RevokeRight.java

```diff
@@ -43,6 +48,17 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
 
         RightCommand.revokeRight(Provisioning.getInstance(), getAuthenticatedAccount(zsc), rrReq.getTarget(),
                                 rrReq.getGrantee(), rrReq.getRight().getValue(), rightModifier);
+        EffectiveRightsTargetSelector erTargSel = rrReq.getTarget();
+        if (com.zimbra.soap.type.TargetType.global == erTargSel.getType()) {
+            CacheSelector cacheSel = new CacheSelector(true /* allServers */, CacheEntryType.globalgrant.toString());
+            FlushCacheRequest fcReq = new FlushCacheRequest(cacheSel);
+            try {
+                FlushCache.doFlushCache(this, context, fcReq);
+            } catch (ServiceException se) {
+                ZimbraLog.acl.info("Problem flushing acl cache for global %s/%s after revoking rights",
```

Comment:
warning log would be a better option here

Reply:
It's also "info" level in the change further above, so should probably change both to be consistent, if changing.

Reply:
done